util/test/chpl_launchcmd.py

```diff
@@ -115,6 +115,17 @@ def full_test_command(self):
         :returns: command to run in qsub with changedir call
         """
         full_test_command = ['cd', '$PBS_O_WORKDIR', '&&']
+
+        # If the first argument of the test command is a file (it should
+        # always be the executable), then add a "test -f ./execname" call
+        # before running the command. This works around some potential nfs
+        # configuration issues that can happen when running from lustre
+        # mounted over nfs.
+        if os.path.exists(self.test_command[0]):
+            logging.debug('Adding "test -f {0}" to launcher command.'.format(
+                self.test_command[0]))
+            full_test_command += ['test', '-f', self.test_command[0], '&&']
```

Comment:
Oh wait.. you're checking that it exists above

So the file already exists so the only way test -f would fail is if the file was removed between this code and when the job is launched (which is really unlikely.) I guess I'm just confused about how this fixes / skirts around the issue. Is this basically touch the file on the compute node? And if so would it be more obvious (and still work) to do a touch instead of test -f?